src/router/OSRMRouter.js

```diff
@@ -0,0 +1,115 @@
+import {fromLonLat, toLonLat} from 'ol/proj.js';
+
+/**
+ * @typedef {import("ol/proj").ProjectionLike} ProjectionLike
+ * @typedef {import('ol/geom/LineString').default} LineString
+ * @typedef {import('ol/geom/Point').default} Point
+ */
+
+
+/**
+ * @typedef {Object} Options
+ * @property {ProjectionLike} mapProjection
+ * @property {string} options.url The URL profile prefix to use, see *_PROFILE_URL.
+ * @property {string} options.extraParams Parameters like access token.
+ * @property {number} options.radius
+ */
+
+
+/**
+ * @const {string}
+ */
+const OSM_CH_ROUTED_FOOT_PROFILE_URL = 'https://routing.osm.ch/routed-foot/route/v1/driving';
```

Comment:
```suggestion
export const OSM_CH_ROUTED_FOOT_PROFILE_URL = 'https://routing.osm.ch/routed-foot/route/v1/driving';
```
and remove export at the bottom of the page